### PR TITLE
Concert Bot Rules

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- Added RulePolicy to config, removed fallback & mapping policies
- Implemented chitchat & nlu_fallback intents as rules

### CODE REVIEW

**Comment by akelad** [comment]
@b-quachtran thanks for submitting this - the same goes here as in Arjaans PR, the stories should also be converted to yaml and so should the nlu training data (if there is any)

**Comment by akelad** [comment]
@b-quachtran what about the stories for searching venues/concerts?

**Comment by b-quachtran** [comment]
@akelad Oops sorry .gitignore was set to ignore changes in the concertbot/data folder -_-

File: examples/concertbot/data/nlu.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+nlu:
+  - intent: greet
+    examples: |
+      hi
```
Comment (akelad) [inline examples/concertbot/data/nlu.yaml:4]: can you add the `-` prefix here? Like in Arjaans PR: https://github.com/RasaHQ/rasa/pull/6229/files#diff-ed593145bcc162429c6f89a64f2fcc0dR5

File: examples/concertbot/data/rules.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+rules:
+  - rule: fallback
+    steps:
+      - ...
+      - intent: nlu_fallback
```
Comment (akelad) [inline examples/concertbot/data/rules.yaml:5]: did you test this works? I personally have no idea how the fallback works in the new setup. Doesn't the threshold need to be defined somewhere?

Comment (b-quachtran) [reply examples/concertbot/data/rules.yaml:5]: I implemented it based on Tobias' doc https://github.com/RasaHQ/rasa/pull/6195/files#diff-9bbb964215f342154b3f976a61d47d2cR43. Just updated the config file and it's working now

**Pull request merged**